src/FileServiceManager/FileService/Helpers/RESTException.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Security.Permissions;
+using System.Net;
+
+namespace FileService.Helpers
+{
+    public class RESTException : Exception
```

Comment:
didn't saw the usage of this class, is it for future use ?

Reply:
I supposed to use it in future as one type for all type exceptions. We can map all exceptions to this exception type and let the client to "parse" our exception. Anyway, currently I remove it till the last decision.